experimenter/experimenter/experiments/models.py

```diff
@@ -812,6 +812,9 @@ def clone(self, name, user, rollout_branch_slug=None):
 
         return cloned
 
+    def get_changelogs(self):
+        return list(self.changes.all())    
+
```

Comment:
This doesn't provide any benefit since we're not pulling it into a variable.

Reply:
oh yes this is just for now temporarily, to make sure we have a function that returns something related to changelogs, in the future PR he will do processing and everything